eng/pipelines/update-azure-linux-pipeline.yml

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# For info about runtime parameters, see https://github.com/microsoft/go-infra/blob/main/docs/pipeline-yml-style.md#runtime-parameters
+parameters:
+  # If someone's manually queueing this build, let them skip the approval stage. It takes time to
+  # acquire a serverless agent to present the approval step, so the dev has to pay attention and
+  # catch it. Instead, let them approve it ahead of time by skipping it. The default is always used
+  # by automation, so this is only used for dev builds and retries.
+  - name: approveAheadOfTime
```

Comment:
I don't think approveAheadOfTime is needed for this pipeline. `release-go-images-pipeline.yml` has an approval step because once that pipeline runs, it directly changes the production images in MAR. The approval means the release owner/dev has a chance to check things over first. (Specifically: did innerloop tests pass?)

This pipeline submits a PR, but it doesn't directly change any production assets. The change has to go through PR, and the release owner/dev can close the PR (or simply not merge it) if there's a problem that comes up.

I think deleting this parameter, everything in `- ${{ if eq(parameters.approveAheadOfTime, false) }}:`, and `retryInstructionsFlags` is fine, to simplify the pipeline.

Reply:
fixed